apps/opencs/view/doc/viewmanager.cpp

```diff
@@ -60,6 +61,40 @@ CSVDoc::ViewManager::ViewManager (CSMDoc::DocumentManager& documentManager)
     mDelegateFactories->add (CSMWorld::ColumnBase::Display_RefRecordType,
         new CSVWorld::IdTypeDelegateFactory());
 
+    // Columns with QLineEdit editor
+    static const CSMWorld::ColumnBase::Display sIdCompletionColumns[] =
+    {
+        CSMWorld::ColumnBase::Display_Cell,
+        CSMWorld::ColumnBase::Display_Class,
+        CSMWorld::ColumnBase::Display_Creature,
+        CSMWorld::ColumnBase::Display_Faction,
+        CSMWorld::ColumnBase::Display_String,
```

Comment:
https://github.com/OpenMW/openmw/blob/master/apps/opencs/model/world/columnimp.hpp#L736-L739
https://github.com/OpenMW/openmw/blob/master/apps/opencs/model/world/columnimp.hpp#L1269-L1273
https://github.com/OpenMW/openmw/blob/master/apps/opencs/model/world/columnimp.hpp#L1830-L1834

Do this columns contain ID information or not?